### DISCUSSION TITLE
Detect duplicated entries

### DISCUSSION THREAD

**Discussion by skr3ch** [mtotschnig/MyExpenses#1506] [Ideas]
Could be useful to highlight duplicated/triplicated entries to avoid wrong sum (for example in the monthly amounts).
Don't know if the duplication detection could be based on the couple: entry amount + notes + date

**Comment by mtotschnig** [comment]
I'd be interested to learn more about the use case. How do you end up with these duplicates, what is there source?

**Reply by skr3ch** [reply]
I try to explain. Personally, I have an account corresponding to my monthly cashflow. So I have transactions for my incomes and expenses. Sometimes it can happen that I know an expense several days before and I add it, especially if it has a big amount, it helps me to keep my balance under control. But several days after, when I really spend that money, I should always search if I already added the expense to my account. (And there, a search bar cross accounts could be another idea)
